tests/dir.props

```diff
@@ -132,5 +134,18 @@
   <PropertyGroup>
     <CLRTestPriorityToBuild>0</CLRTestPriorityToBuild>
   </PropertyGroup>
+
+  <!-- Where to put a "testhost" for running corefx tests -->
+  <PropertyGroup>
+    <!-- Specify the version number used in the testhost directory. This should be $(ProductVersion), to match what corefx does,
+         but the current set of tests we're using still use the hard-coded 9.9.9.
+    -->
+    <TestHostVersion>9.9.9</TestHostVersion>
+    <!-- <TestHostVersion>$(ProductVersion)</TestHostVersion> -->
```

Comment:
I guess this should now be replaced.

Reply:
Yes. This can be a follow-up change; I want to get this in with minimal additional changes currently, and I think it works with this due to roll-forward. I opened https://github.com/dotnet/coreclr/issues/25394 to track.

Reply:
I'm not a fan of keeping the 9.9.9 version here, this is VERY fragile. I would prefer to see this change as part of this PR. I'm not sure either how long I will keep the roll-forward policy in the test scripts.

Reply:
> I'm not a fan of keeping the 9.9.9 version here, this is VERY fragile.

It can't be that fragile given that we have run with this fake version for like two years.

I agree with Bruce that it is get this PR in without attaching more cleanups to it.

Reply:
It turns out I had to run a new CI pass anyway, so I had gone ahead and picked up all the "TO DO" cleanups identified by the code reviews, including this.